tests/Ormolu/PrinterSpec.hs

```diff
@@ -49,6 +49,8 @@ checkExample srcPath' = it (fromRelFile srcPath' ++ " works") . withNiceExceptio
               Set.fromList
                 [ "base",
                   "esqueleto",
+                  "hspec",
+                  "hspec-expectations",
```

Comment:
Just to double-check: The `hspec-expectations` would not be necessary with https://github.com/tweag/ormolu/issues/1037, right?

Reply:
Yes, that's correct.